docs/docs/architecture/attestation.md

```diff
@@ -220,6 +220,46 @@ The latter means that the value can be generated offline and compared to the one
 </tabItem>
 </tabs>
 
+### CVM verification
+
+To verify runtime measurements, a chain of trust from the CVM technology to the interface providing the measurements has to be established.
```

Comment:
```suggestion
To verify the integrity of the received runtime measurements, a chain of trust from the CVM technology to the interface providing the measurements has to be established.
```
Should we be more explicit here? "verify runtime measurements" could be confused with "comparing the actual measurements against the expected ones".